Neuro/Minigames/Solvers/ChartCourseSolver.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+using UnityEngine;
+
+namespace Neuro.Minigames.Solvers;
+
+[MinigameSolver(typeof(CourseMinigame))]
+public class ChartCourseSolver : MinigameSolver<CourseMinigame>
```

Comment:
Not working

![image](https://user-images.githubusercontent.com/32238504/230696934-57bcd2dc-6625-4616-8e20-ae4b49f77c53.png)